src/components/SearchResults/SearchError.tsx

```diff
@@ -0,0 +1,93 @@
+import { Heading, Flex, Link, Text } from "@nypl/design-system-react-components"
+import type { HTTPStatusCode } from "../../types/appTypes"
+import { appConfig } from "../../config/config"
+import { SITE_NAME } from "../../config/constants"
+import RCHead from "../Head/RCHead"
+import Layout from "../Layout/Layout"
+import ExternalLink from "../Links/ExternalLink/ExternalLink"
+import { useContext } from "react"
+import { FeedbackContext } from "../../context/FeedbackContext"
+
+type SearchErrorProps = {
+  errorStatus: HTTPStatusCode
+}
+
+export default function SearchError({ errorStatus }: SearchErrorProps) {
+  const metadataTitle = `${errorStatus} | ${SITE_NAME}`
+  const { onOpen } = useContext(FeedbackContext)
+  const errorContent = (status) => {
+    switch (status) {
+      case 404:
+        return (
+          <>
+            <Heading level="h3" tabIndex={-1} id="search-results-heading">
+              No results found.
+            </Heading>
+            <p>
+              Try a different search, or search our{" "}
+              <ExternalLink href={appConfig.urls.legacyCatalog}>
+                Legacy Catalog
+              </ExternalLink>{" "}
+              for research materials.
+            </p>
+          </>
+        )
+      case 500:
+        return (
+          <>
+            <Heading level="h3" tabIndex={-1} id="search-results-heading">
+              Something went wrong on our end.
+            </Heading>
+            <Text marginBottom="0">
```

Comment:
FYI, `Text`'s `noSpace` prop does the same as `marginBottom="0"`, but this is not a blocker.